build.gradle

```diff
@@ -48,10 +50,27 @@ dependencies {
         compile 'com.google.firebase:firebase-messaging:11.+'
     } else {
         compile fileTree(include: 'gcm.jar', dir: 'libs')
-        compile 'com.android.support:support-v4:26.0.1'
     }
+    compile 'com.google.firebase:firebase-iid:11.+'
```

Comment:
actually this should be a more specific version, as should the one just above, for firebase

Reply:
Updated: set to 11.8.0